src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -50,5 +50,7 @@ public static class EnvironmentSettingNames
 
         public const string LinuxAzureAppServiceStorage = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
         public const string CoreToolsEnvironment = "FUNCTIONS_CORETOOLS_ENVIRONMENT";
+
+        public const string AlternateCdnUri = "FUNCTIONS_ALTERNATE_CDN_URI";
```

Comment:
Why is this called AlternateUri? is this a secondary?

Reply:
This is to help development and to help us provide secondary cdn link incase the first one is busted for some reason.